waspc/data/Generator/templates/sdk/wasp/server/oauth/providers/github.ts

```diff
@@ -0,0 +1,17 @@
+{{={= =}=}}
+import { GitHub  } from "arctic";
+
+import { ensureEnvVarsForProvider } from "../env.js";
+
+export const id = "{= providerId =}";
+export const displayName = "{= displayName =}";
+
+const env = ensureEnvVarsForProvider(
+  ["GITHUB_CLIENT_ID", "GITHUB_CLIENT_SECRET"],
```

Comment:
I don't want to abstract this code away since it's pretty simple and there are subtle differences between OAuth provider initialisation. This is easy to read and easy to maintain IMHO.

Reply:
I'm against calling this function with a spread operator while mapping over env vars, it feels too complex even with just that. I've written enough of the auth related code to get bitten by not keeping things dead simple. This is very basic Lucia OAuth client invocation and I'd like to keep it like that.